pkgs/development/python-modules/hfst-python/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, hfst
+, icu
+, swig
+, python3Packages
+}:
+
+python3Packages.buildPythonPackage {
+  pname = "hfst-python";
+  inherit (hfst) version src;
+
+  sourceRoot = "source/python";
+
+  postPatch = ''
+    # omorfi-python looks for 'hfst' Python package
+    sed -i 's/libhfst_swig/hfst/' setup.py;
+  '';
+
+  nativeBuildInputs = [
+    swig
+  ];
+
+  buildInputs = [
+    icu
+    hfst
+  ];
+
+  setupPyBuildFlags = [
+    "--inplace"
+  ];
```

Comment:
Why do we need this? It is rather unusual that this is required

Reply:
Without that, building omorfi fails:
```
checking python module: libhfst... no
configure: error: failed to find required module libhfst
```